### PR TITLE
chore: sync autoware.rviz

### PR DESCRIPTION
**Pull request by shmpwk**
Signed-off-by: Shumpei Wakabayashi <shumpei.wakabayashi@tier4.jp>

## PR Type

- New Feature

## Description

sync awf/ autoware.rviz -> tier4/ autoware.rviz

## Review Procedure

Check if there is no problem to sync

## Remarks
Current difference between [awf/rviz](https://github.com/autowarefoundation/autoware_launch/blob/5f34aa8524cd0e36d17e08fc9fd2311c49466902/autoware_launch/rviz/autoware.rviz) (red color) and [tier4/rviz](https://github.com/tier4/autoware_launch/blob/dee0e66f04701b71222aefa7a214642bfd367810/autoware_launch/rviz/autoware.rviz) (green color)

Screenshot from 2022-07-15 11-07-17
Screenshot from 2022-07-15 11-07-44
Screenshot from 2022-07-15 11-07-54
Screenshot from 2022-07-15 11-08-09
Screenshot from 2022-07-15 11-08-23
Screenshot from 2022-07-15 11-08-33
Screenshot from 2022-07-15 11-08-44
Screenshot from 2022-07-15 11-13-26
Screenshot from 2022-07-15 11-13-37
Screenshot from 2022-07-15 11-13-50

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR**: Required to pass before the merge.
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: .github/sync-param-files.yaml [hunk 1]
```diff
@@ -28,3 +28,8 @@
       dest: system_launch/config/system_monitor/ntp_monitor.param.yaml
     - source: system/system_monitor/config/process_monitor.param.yaml
       dest: system_launch/config/system_monitor/process_monitor.param.yaml
+
+- repository: autowarefoundation/autoware
+  files:
+    - source: autoware_launch/autoware_launch/rviz/autoware.rviz
+      dest: autoware_launch/autoware_launch/rviz/autoware.rviz
```
Comment (kenji-miyake) [inline .github/sync-param-files.yaml]: FYI: I remember you can omit the `dest` value if it's the same as the `source`.

```suggestion
```

**Review by kenji-miyake — COMMENTED** [review]
If there are some differences now, I guess you have to consider updating the `awf/rviz` first.

**Comment by tkimura4** [comment]
Status:
This PR needs to be merged, but requires a rebase before merging.

File: .github/sync-param-files.yaml [hunk 2]
```diff
@@ -28,3 +28,8 @@
       dest: system_launch/config/system_monitor/ntp_monitor.param.yaml
     - source: system/system_monitor/config/process_monitor.param.yaml
       dest: system_launch/config/system_monitor/process_monitor.param.yaml
+
+- repository: autowarefoundation/autoware
```
Comment (kenji-miyake) [inline .github/sync-param-files.yaml]: By the way, is here wrong?

```suggestion
- repository: autowarefoundation/autoware_launch
```

Comment (shmpwk) [reply .github/sync-param-files.yaml]: Yes it seems wrong

**Comment by shmpwk** [comment]
I rebased, resolved conflict and reflected miyake-san comments.

**Comment by shmpwk** [comment]
Close this because of duplicated PR  https://github.com/tier4/autoware_launch/pull/546/files

**Pull request closed without merging**